os/binfmt/libelf/libelf_symbols.c

```diff
@@ -221,7 +224,7 @@ void elf_readsymtab(FAR struct elf_loadinfo_s *loadinfo)
 		return;
 	}
 
-	if (elf_read(loadinfo, loadinfo->symtab, symtab->sh_size, symtab->sh_offset) < 0) {
+	if (elf_read(loadinfo, (FAR uint8_t *)loadinfo->symtab, symtab->sh_size, symtab->sh_offset) < 0) {
```

Comment:
do we need to add similar typecast at line 111 -> (FAR uint8_t *)loadinfo->strtab

Reply:
Yeah it requires. I was modifying. It will be updated more. libfmt_loadbinary.c and binfmt_execmodule.c left.